drivers/SCsub

```diff
@@ -14,6 +14,8 @@ SConscript("coreaudio/SCsub")
 SConscript("pulseaudio/SCsub")
 if env["platform"] == "windows":
     SConscript("wasapi/SCsub")
+    if not env.msvc:
```

Comment:
Would it be in scope to expand to all the platforms Godot supports? At least Linux and Mac or should it be in a different pr?